templates/angular/set-contract-name.js

```diff
@@ -0,0 +1,16 @@
+const { writeFileSync } = require ('fs')
+require('dotenv').config({ path: './neardev/dev-account.env' })
+const colors = require('colors')
+
+const targetPath = './src/contract-name.js'
+const contractNameFileContent = `module.exports = '${process.env.CONTRACT_NAME || 'near-blank-project'}'`
```

Comment:
We cannot access the environment variables in angular. If we try to get the value of `process.env.CONTRACT_NAME`, then instead of the name of the contract we will get `undefined`.

https://stackoverflow.com/questions/49496438/use-process-env-in-angular-5-environment#:~:text=the%20Angular%20code.-,process.,truly%20needs%20to%20be%20dynamic.

To create similar variables for different build modes, I could use `src/environments/environment.ts`, but this is an unnecessary complication, since the developer will supplement this file with his own variables, and `set-contract-name.ts` will need to pointwise change the value of the `contractName` key (using `replace-in-files`).

Reply:
Gotcha, that's unfortunate. For my own understanding, is this `set-contract-name` a common workaround or your own invention?

Reply:
This solution is based on [the article](https://medium.com/javascript-in-plain-english/setup-dotenv-to-access-environment-variables-in-angular-9-f06c6ffb86c0), but the target file is different. This is done so that the developer can still work with `environment*.ts` files, set his own variables, and his changes will not be overwritten after `set-contract-name` processing.